src/components/m-datepicker/index.scss

```diff
@@ -49,29 +50,36 @@ $datepicker-padding-left-native-dropdown: 8px !default;
     }
   }
 
-  .m-dropdown__select-wrap {
-    display: inline-block;
-  }
-
   .m-dropdown__select {
     @include dropdown-select($indent: 10px, $height: 40px);
   }
 
   .m-dropdown__select-icon {
     @include dropdown-select-icon($indent: 10px, $height: 40px);
+    width: 35px;
```

Comment:
@jackblackCH 
@MarekLacoAXA 
May it fits better to add `width` arg to [`dropdown-select-icon`](https://github.com/axa-ch/patterns-library/blob/dbaf0823e7d5de376d2943c135d0c8d80af1b96f/src/components/m-dropdown/mixins/dropdown.scss#L127) mixin

Reply:
anyway not plain `px` as it is in our StyleLint and StyleLint is now in our [CoC](https://github.com/axa-ch/patterns-library/blob/develop/CODE_OF_CONDUCT.md)

Reply:
REF: _We make sure that our code respects ESlint, StyleLint and EditorConfig settings. We can challenge the setting but wont add our own code style if it goes against linting files._